test/quantization/test_quantize_fx.py

```diff
@@ -757,3 +768,177 @@ def forward(self, x):
             quantized,
             expected_node_occurrence=count_check,
             expected_node_list=order_check)
+
+class TestQuantizeFxModels(QuantizationTestCase):
+    @skipIfNoTorchVision
+    def test_torchvision(self):
+        from torchvision import models
```

Comment:
are there cross-repo compatibility concerns to think through here?  AFAIK torchvision doesn't fully enforce a way to make breaking API changes in a BC/FC compatible way (it's more of an honor system where the promise it that it is guaranteed on major releases, but not on every PR).

Reply:
I don't feel we are relying on something that will change in torchvision if that is what you are worrying about

Reply:
I also saw in https://github.com/pytorch/pytorch/blob/master/.circleci/config.yml#L1825 that it is using
```
pip install torchvision
```
so it should just be installing major releases

Reply:
sg, thanks for explaining!